### PR TITLE
Frontend team: Merge in changes from sprint 1

### PR DESCRIPTION
**Pull request by daisycrego**

### CODE REVIEW

File: mercury/tests/selenium_tests/test_create_event.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    def test_create_event(self):
+        # login with testcode
+        self.login_test_code()
+
+        # create a new event
+        self.open(reverse("mercury:event"))
+        self.wd.find_element_by_id("id_event_name").send_keys("event1")
+        self.wd.find_element_by_id("id_event_location").send_keys("New York")
+        self.wd.find_element_by_id("post-event-date").send_keys("2020-03-03")
+        self.wd.find_element_by_id("post-event-comments").send_keys("event1 comment")
```
Comment (Ricardo-1894) [inline mercury/tests/selenium_tests/test_create_event.py]: Hi I think this part is not corresponding to the current database schema

File: mercury/views/events.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            post_event_name = request.POST.get("event_name")
+            post_event_location = request.POST.get("event_location")
+            post_event_date = request.POST.get("date")
+            post_event_comments = request.POST.get("comments")
+            event_data = Event(
+                event_name=post_event_name,
+                event_location=post_event_location,
+                date=post_event_date,
+                comments=post_event_comments,
+            )
```
Comment (Ricardo-1894) [inline mercury/views/events.py]: Same as what I mentioned, this part also does not correspond to current database design

File: mercury/tests/selenium_tests/test_create_event.py (hunk 1 above)
Comment (daisycrego) [reply mercury/tests/selenium_tests/test_create_event.py]: @Ricardo-1894 yes indeed - do you know when those changes to the models will get into the master?

Comment (Ricardo-1894) [reply mercury/tests/selenium_tests/test_create_event.py]: @daisycrego You can find these changes in the `[backend-team](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/tree/backend-team)` branch now, we will probably merge changes into master later.

Comment (daisycrego) [reply mercury/tests/selenium_tests/test_create_event.py]: @Ricardo-1894 awesome! we will close this PR for now

**Pull request closed without merging**